vnext/Desktop.UnitTests/UtilsTest.cpp

```diff
@@ -268,6 +268,79 @@ TEST_CLASS(UtilsTest)
   }
 
 #pragma endregion Base64 Tests
+
+#pragma region string_format Tests
+
+  TEST_METHOD(UtilsTest_StringFormat_Simple)
+  {
+    auto result = Microsoft::React::string_format("Hello, %s!", "World");
```

Comment:
Please avoid using `auto` unless you absolutely must use it.
Having the `std::string` here in the new code is much more readable.

Reply:
curious, why avoid auto? I've never actually known what is better style